feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -377,27 +383,27 @@ func TestVrfPolicy(t *testing.T) {
 		{
 			desc:      "Match IPv4",
 			policy:    "match-ipv4",
-			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10"},
+			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "Ipv4Vlan20", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20"},
 		},
 		{
 			desc:      "Match IPinIP",
 			policy:    "match-ipip",
-			passFlows: []string{"IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10"},
+			passFlows: []string{"IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20"},
```

Comment:
Wouldnt this fail the test. Requirement says,

*   Matching of IPinIP protocol - to network-instance 10 for all input IPinIP         
                No received IPv4 or IPv6 in network-instance 10.

No mention of traffic in VLAN20 as pass criteria

Reply:
Latest changes pushed in. please check. Thanks.